packages/story-editor/src/components/videoTrim/trimmerComponents.js

```diff
@@ -49,6 +50,7 @@ export const Rail = styled.div`
   border-radius: 4px;
   border: 1px solid ${({ theme }) => theme.colors.standard.white};
   background-color: ${({ theme }) => theme.colors.standard.white};
+  ${elementWithFlip}
```

Comment:
Is this necessary? Isn't the rail correctly rendered? I guess this doesn't do anything because we don't pass a `flip` property to the `<Rail />` element?

Reply:
Yes, this is unnecessary, would flip the entire canvas instead of the individual frames. Removing